src/main/java/crazypants/enderio/machine/soul/SoulBinderSpawnerRecipe.java

```diff
@@ -21,26 +23,26 @@ public SoulBinderSpawnerRecipe() {
 
   @Override
   protected ItemStack getOutputStack(ItemStack input, CapturedMob mobType) {
-    return mobType.toStack(itemBrokenSpawner.getItem(), input.getMetadata(), 1);
+    return mobType.toStack(ModObject.itemBrokenSpawner.getItem(), input.getMetadata(), 1);
   }
 
   @Override
   protected boolean isValidInputSoul(CapturedMob mobType) {
-    return getSupportedSouls().contains(mobType.getEntityName()) && !BlockPoweredSpawner.isBlackListed(mobType.getEntityName());
+    return getSupportedSouls().contains(mobType.getEntityName()) && !BlockPoweredSpawner.isBlackListed(mobType.getEntityName().toString());
```

Comment:
Perhaps `isBlackListed` should be rewritten to take an RL? Might be outside the scope of this PR